lms/static/styles/_mixins.scss

```diff
@@ -1,3 +1,6 @@
+@use "variables" as *;
```

Comment:
From the PR description:

> Converted the @ use "../variables" to use the wildcard * namespace as its our convention for variables instead of prefixing the namespace when used.

Well, we didn't have a namespace convention before because the SASS language provided no choice.

A downside of wildcard imports is that, as well as potential conflicts, it makes it less obvious where a variable comes from.

Using the default namespace for variables would make usage perhaps a bit verbose (`variables.$small-font-size`). The [SASS docs](https://sass-lang.com/documentation/at-rules/use) seem to suggest a convention of picking a short alias for common imports. Perhaps we should do something like:

```scss
@use "variables" as v;

...

.some-class {
  font-size: v.$small-font-size;
}
```

Thoughts?

Reply:
I was thinking of using "v" too, just didn't know if we wanted to use namespaces, but yes, I can see the value.